### DISCUSSION TITLE
[indexer] Get ready to 1.20.0

### DISCUSSION THREAD

**Discussion by khorolets** [near/nearcore#4386] [Node Public Interfaces]
*update:* Dates of releases changed, **recommended commit hash has changed**
*update:* Dates of releases changed (release is rescheduled for one week further)

Jul 5, 2021 nearcore 1.20.0 release candidate will be deployed to testnet

Jul 12, 2021 nearcore 1.20.0 is going to be deployed to mainnet

## Getting ready

Please, get your indexer prepared. Follow [recommendation from NEAR Indexer Framework 0.9.2 release notes](https://github.com/near/nearcore/discussions/4385)

### Quick info

**Commit hash** [9655ff694e8c38277b184ca42dc3478bbd4fd86a](https://github.com/near/nearcore/commit/9655ff694e8c38277b184ca42dc3478bbd4fd86a)
**Expected issues** Underlying dependencies versions might cause compilation error. To resolve it just replace the content of your `Cargo.lock` with the content from the same file from the recommended commit. [Go to raw Cargo.lock](https://raw.githubusercontent.com/near/nearcore/9655ff694e8c38277b184ca42dc3478bbd4fd86a/Cargo.lock)
**Reference** have a look at how [we made it on NEAR Indexer for Explorer](https://github.com/near/near-indexer-for-explorer/pull/131/commits/f6b3a3120c2c6f7b9a42229de0ceebd18cfe417d)

Feel free to leave your questions in the comments below.

**Comment by maxzaver** [comment]
> Next week Jun 22, 2021 nearcore 1.20.0 is going to be deployed to mainnet

@khorolets Is this correct?

**Comment by khorolets** [comment]
> > Next week Jun 22, 2021 nearcore 1.20.0 is going to be deployed to mainnet
> 
> 
> 
> @khorolets Is this correct?

No, it's not! So I am the source of misleading. Thank you! I've fixed :(

**Comment by khorolets** [comment]
Heads up! Release dates have changed. I've edited the original post.
